data_transfer/devices/thinkfast.py

```diff
@@ -66,17 +66,14 @@ def format_record(self, raw_rec: Dict, participant: Participant) -> Record:
             end_wear=raw_rec["itemGroups"][0]["endTime"],
             is_downloaded=True,
         )
+        # remove the 'user' field which contains
+        # clinician name and email and therefore may be a GDPR concern
+        raw_rec.pop("users")
+
+        is_cantab = raw_rec["itemGroups"][0]["items"][0]["measureCode"] == "SWMTE"
+        tfa_type = "CANTAB" if is_cantab else "ThinkFAST"
+        new_record.meta = {"tfa_type": tfa_type, "full_data": raw_rec}
```

Comment:
LGTM! Great solution and tested on DMP test dataset and can confirm that the `"full_data"` now includes timestamps and other relevant data similar to CANTAB.